### DISCUSSION TITLE
How to use on macos?

### DISCUSSION THREAD

**Discussion by mredd** [wmbusmeters/wmbusmeters#1337] [Q&A] [answered]
Hello,

I've a nooelec Smart SDR which works perfectly at my MacBook. However, I've no idea how to get it to work with the wmbusmeters application. Compilation was not a problem, worked out of the box.

However, when I try to start `wmbusmeters rtlwmbus`, I get this error message:
```
Warning! Auto scan has found an rtl_sdr dongle, but you have no rtl_wmbus in the path!
[ALARM SpecifiedDeviceNotFound] the device rtlwmbus is not working
No wmbus device detected, waiting for a device to be plugged in.
```

I have the rtlsdr lib installed via `brew install librtlsdr`, but that does not include the missing `rtl_wmbus` tool. Which additional tool do I need to install to make wmbusmeters work under macos?

**Comment by asbachb** [comment, marked as answer]
Have you checked/installed `rlt_wmbus`: https://github.com/xaelsouth/rtl-wmbus ?

**Reply by mredd** [reply]
Thank you for the hint @asbachb. I did not consider it due to the heavy emphasize on being an experimental platform.

And sorry for the late reply. I was busy with work and could not work further on my toy project. 😞
